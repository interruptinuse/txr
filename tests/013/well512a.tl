(let ((rs (make-random-state #(#x01010101 #x11111111 #x22222222 #x33333333
                               #x44444444 #x55555555 #x66666666 #x77777777
                               #x88888888 #x99999999 #xAAAAAAAA #xBBBBBBBB
                               #xCCCCCCCC #xDDDDDDDD #xEEEEEEEE #xFFFFFFFF
                               0)))
      (mod (expt 2 32)))
  (for ((i 0)) ((< i 16)) ((inc i))
    (pprinl (rand mod rs))))
