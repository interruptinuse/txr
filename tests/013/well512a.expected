419341993
1818026447
3020103184
3141358934
187862696
512541134
1416267688
4256491982
731233432
1852014262
1248293699
206481201
750755084
1094637740
2806919360
2752871950
